R/taxonomy.R

```diff
@@ -688,18 +698,18 @@ setMethod("mapTaxonomy", signature = c(x = "SummarizedExperiment"),
 }
 
 #' @importFrom SummarizedExperiment rowData
-.get_tax_groups <- function(x, col, onRankOnly = FALSE, ...){
+.get_tax_groups <- function(x, col, disable.taxonomy = FALSE, ...){
```

Comment:
We moved onRankOnly to be hidden. However, we cannot probably just remove it (even though there is no really use for the option). This modification removes it especially because disable.taxonomy is not documented anywhere (which is good)

--> add also onRankOnly here

Reply:
ACK.